### DISCUSSION TITLE
change identifier

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14325] [How to] [answered]
in vtiger we could do like that:

 include_once('vtlib/Vtiger/Module.php');
 include_once('vtlib/Vtiger/Field.php');

 $module = Vtiger_Module::getInstance('BookingRegister');
 $field = Vtiger_Field::getInstance('type_of_booking_services', $module);

 $module->unsetEntityIdentifier();
 $module->setEntityIdentifier($field);

How it is now in YF?

**Comment by rskrzypczak** [comment, marked as answer]
https://gitdeveloper.yetiforce.com/index.php?module=Search&parent=Settings&view=Index
![image](https://user-images.githubusercontent.com/10221999/110075102-942b0580-7d82-11eb-8501-f34412185ca4.png)
